blog/2022-08-04-gradients-button-textfield-styles.md

```diff
@@ -245,58 +240,56 @@ An example of styled `TextField` with `max_length` and `capitalization`:
 <img src="/img/blog/gradients/styled-textfield.gif" className="screenshot-50" />
 
 ```python
-import flet
-from flet import Page, TextField, colors
+import flet as ft
 
-def main(page: Page):
+def main(page: ft.Page):
     page.padding = 50
     page.add(
-        TextField(
+        ft.TextField(
             text_size=30,
-            cursor_color=colors.RED,
-            selection_color=colors.YELLOW,
-            color=colors.PINK,
-            bgcolor=colors.BLACK26,
+            cursor_color= ft.colors.RED,
+            selection_color= ft.colors.YELLOW,
+            color= ft.colors.PINK,
```

Comment:
There are still inconsistencies like this `= `across multiple samples. I understand it's a lot, but could could you fix all kwargs to don't have spaces around `=`. It's easier to spot such places before merging a PR. Thank you!

Reply:
Ok, I will fix them, I will go over the whole docs again and fix those issues.

Reply:
I think things are good now, do you mind giving it a second look?